source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -257,6 +291,44 @@ export abstract class CardElement {
         // Does nothing in base implementation
     }
 
+    handleOverflow(maxHeight: number) {
```

Comment:
Too bad this is public... I know why, but it isn't great. When I hit those situations, I made the methods protected or private and I call them using the object["methodName"] pattern, which isn't great but a good approximation of what the "internal" modifier would be.